File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/user/UserLoginMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.user;

public class UserLoginMessage {

	public static final String TYPE = "UserLoginMessage";
	private String username;

	public UserLoginMessage() {
	}

	public UserLoginMessage(String username) {
		this.username = username;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}
}

File: client/Client/Network/Messages/Session/UserLoginMessage.cs
namespace LoadRunnerClient.Messages
{
    public class UserLoginMessage
    {
        public const string TYPE = "UserLoginMessage";

        /// Author Florian Ortmann
		public UserLoginMessage() { }

        public UserLoginMessage(string username)
        {
            this._username = username;
        }

        private string _username;

        public string username
        {
            get => _username;
            set => _username = value;
        }
    }
}